draft-packages/avatar/KaizenDraft/Avatar/Avatar.tsx

```diff
@@ -54,10 +54,21 @@ export const Avatar = ({
     "none" | "error" | "loading" | "success"
   >(avatarSrc ? "loading" : "none")
 
+  useEffect(() => {
+    setAvatarState(avatarSrc ? "loading" : "none")
+  }, [avatarSrc])
+
+  const nameHasAlpha = fullName && /[a-zA-Z]+$/.test(fullName)
```

Comment:
I think we need to be really careful here. This pattern will fail on names like `Étienne` or `Nguyễn` which contain diacritic characters.

Reply:
OK wait, I was too hasty 😬  I assumed every character had to match, but you're only looking for the presence of at least one alphabetic char. Oops

Reply:
Yea, had a think about the logic,
It does add some complexity, but yes regarding your comment below. The prompt for this was, sometimes in our platform, users don't have names, they have numbers, but they live in the name field. So we either, do a check for a numeric name _outside_ this component, and just not pass in anything, which adds complexity to the implementation, or we just makes sure there's at least an alpha character, which would make implementation easier.

Open to your Kaizen-minded suggestion. This really only covers us for fully numeric strings (which is the case I am fixing for), and not a combination of the 2, but I'm not sure that's a case we need to cover for now.

That said, an approach that isn't tied to 'Culture Amp' as a platform would make sense. 
Maybe just a prop called `disableInitials`, and maybe just throw in the check upon implementation? Keen to discuss.

Prompt for this work is here: https://cultureamp.atlassian.net/browse/TU-342

Reply:
@sebpearce  - I have had a think, and reworked this to just take a disableInitials prop, and will do the check for the numeric name upon implementation.

This makes more sense from a design system point of view, and will give us more control / less magic.

Reply:
Ah nice one, I didn't think of that and it feels a lot more Kaizen-y. 👌

@ActuallyACat CCing you here since this is an API change you may want to be across.

Reply:
@ActuallyACat Could we confirm that you are OK with this API change?